experimental/packages/opentelemetry-instrumentation/test/common/semver.test.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
```

Comment:
nit: A tweak on the comment to try to make the association with the license/copyright more clear.
I'm copying this existing example: https://github.com/open-telemetry/opentelemetry-js-contrib/blob/4fb610d50e82267c567e5f364cea8f980b403ff7/plugins/node/opentelemetry-instrumentation-aws-sdk/src/propwrap.ts#L17-L21
It is also fine the way you have it.

```suggestion
/*
 * These tests are adapted from semver's tests for `semver.satisfies()`.
 * https://github.com/npm/node-semver/blob/868d4bbe3d318c52544f38d5f9977a1103e924c2/test/functions/satisfies.js
 * License:
 *
```